src/drive/ducks/files/Toolbar.jsx

```diff
@@ -70,7 +71,7 @@ class Toolbar extends React.Component {
           title={t('toolbar.item_more')}
           disabled={disabled}
           className={styles['fil-toolbar-menu']}
-          buttonClassName={styles['fil-toolbar-more-btn']}
+          button={<MoreButton />}
```

Comment:
I'm not sure I understand this change. The `Menu` component already had a `button` prop (apparently?) but it was a boolean. Now you pass in a component but I haven't seen it being used. Did I miss it?

Reply:
It's used here: https://github.com/cozy/cozy-drive/pull/449/files#diff-d660e9521573770d4df9c5faf824ef34R62

Reply:
Aaah yes, missed that. All good then.